src/views/checkout/checkout.php

```diff
@@ -0,0 +1,158 @@
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta
+            name="description"
+            content="Xendit demo store accepting payment with checkout integration"
+        />
+        <link
+            href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600;700&display=swap"
+            rel="stylesheet"
+        />
+        <link rel="stylesheet" type="text/css" href="/css/style.css" />
+        <link rel="stylesheet" type="text/css" href="/css/store.css" />
+        <link rel="icon" href="/images/logo.ico" />
+    </head>
+
+    <body class="preload">
+        <main class="store">
+            <!-- Header -->
+            <div class="header">
+                <div class="wrapper header__wrapper">
+                    <div class="header__site">
+                        <a class="header__link-site" href="/">Xendit Demo</a>
+                    </div>
+                    <div class="header__navigation">
+                        <input
+                            class="header__checkbox"
+                            type="checkbox"
+                            id="header__button-menu"
+                        />
+                        <label
+                            for="header__button-menu"
+                            class="header__button-menu"
+                            ><span class="nav-icon"></span
+                        ></label>
+                        <ul class="header__links">
+                            <li>
+                                <a
+                                    class="header__link-github"
+                                    href="https://github.com/xendit/checkout-demo"
+                                    target="_blank"
+                                    >View Sample Code</a
+                                >
+                            </li>
+                            <li>
+                                <a
+                                    class="header__link-docs"
+                                    href="https://docs.xendit.co/xeninvoice/"
+                                    target="_blank"
+                                    >View Xendit Docs</a
+                                >
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+            <div class="checkout">
+                <div class="wrapper checkout__wrapper">
+                    <!-- Configuration -->
+                    <div class="configure">
+                        <div class="panel-configure">
+                            <h2 class="panel-configure__title">
+                                Welcome to<br />Xendit Demo Store
+                            </h2>
+                            <div class="panel-configure__message">
+                                Xendit is the leading payment gateway for
```

Comment:
Xendit is the leading payment gateway for
                                Indonesia, Philipines and Southeast Asia.